README.ja.md

```diff
@@ -541,7 +543,9 @@ ws.onmessage = function(message) {
 
 この例を実行するには、WebRTC 互換ブラウザを`https://localhost:8443` で開き、ページをオープンするために、HTTPS トラフィックを受け入れます。アプリケーションは、2つの HTML5  `<video>` タグを含む、1つの HTML Web ページで構成されています。1つはローカル Web サーバでキャプチャされたローカル・ストリームと、もう1つはメディア・サーバからクライアントに送信されたリモート・ストリームです。"Start "ボタンをクリックすると、修正されたビデオが右側に表示されます。
 
-![](https://fiware.github.io/tutorials.Media-Streams/img/magic-mirror-screenshot.png)
+[![](https://fiware.github.io/tutorials.Media-Streams/img/magic-mirror-screenshot.png)
+
+[マジック・ミラーのデモ](https://www.youtube.com/watch?v=h84HFkvWGgw "Magic Mirror") を見るにはリンクをクリックしてください。
```

Comment:
Correct - This is the (final) correct method for Linking to the Kurento demo video . 
This is the style to be used for the first link above.